code/game/objects/items/weapons/implants/implant.dm

```diff
@@ -618,3 +618,80 @@ the implant may become unstable and either pre-maturely inject the subject or si
 
 /obj/item/weapon/implant/peace/get_data()
 	return imp_data
+
+
+
+/obj/item/weapon/implant/freedom
+	name = "freedom"
+	desc = "Use this to escape from those evil Red Shirts."
+	_color = "r"
+	var/activation_emote = "chuckle"
+	var/uses = 5
+
+
+/obj/item/weapon/implant/freedom/New()
+	src.activation_emote = pick("blink", "blink_r", "eyebrow", "chuckle", "twitch_s", "frown", "nod", "blush", "giggle", "grin", "groan", "shrug", "smile", "pale", "sniff", "whimper", "wink")
+	..()
+	return
+
+
+/obj/item/weapon/implant/freedom/trigger(emote, mob/living/carbon/source as mob)
+	if (src.uses < 1)
+		return 0
+	if (emote == src.activation_emote)
+		src.uses--
+		to_chat(source, "You feel a faint click.")
+		if (source.handcuffed)
+			source.drop_from_inventory(source.handcuffed)
+		if (source.legcuffed)
+			source.drop_from_inventory(source.legcuffed)
+	return
```

Comment:
return is not necessary here, it should be applied when you want to prematurely end a proc in case of a condition or something, like the return 0 a few lines above